src/components/language.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Select } from 'semantic-ui-react';
+import { compose, withHandlers, pure } from 'recompose';
+import PropTypes from 'prop-types';
+
+const options = [
+  {
+    key: 'ru-RU',
+    value: 'ru-RU',
+    flag: 'ru',
+    text: 'Russian',
+  },
+  {
+    key: 'en-US',
+    value: 'en-US',
+    flag: 'us',
+    text: 'English',
+  },
+];
+
+
+const LanguageChange = ({ onSelect }) => (
+  <Select
+    placeholder="Select language"
+    defaultValue={localStorage.getItem('lang')}
```

Comment:
on first visit the lang is not set in the local storage, so add a default language. in this case we don't need the placeholder anymore

Reply:
the file Name doesn't match the component name